scripts/dps-spring-boot-gradle-plugin/upgrade-latest.bash

```diff
@@ -1,10 +1,13 @@
 #!/usr/bin/env bash
 
-sed -i '' -e 's/spring-boot") version "4.[0-3].[0-9]*\(-beta\)*/spring-boot") version "3.3.12/' build.gradl*
+sed -i '' -e 's/spring-boot") version "[0-9].[0-9].[0-9]*\(-beta\)*/spring-boot") version "4.5.1-beta/' build.gradl*
```

Comment:
guess it should really be parameterised here otherwise we'll keep on having to change it for each release